app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class InvitesController < ApplicationController
+  before_action :set_group
+
+  def create
+    authorize! :manage, @group
+
+    recipient = User.find_by(email: invite_params[:email])
+    invite = @group.invites.build(
+      sender: current_user,
+      recipient: recipient
+    )
+
+    if invite.save
+      flash[:success] = 'Invite sent!'
+    else
+      flash.alert = invite.errors.full_messages.join(', ')
+    end
+
+    redirect_to group_group_members_path(@group)
```

Comment:
that looks pretty ugly, should we use `memberships` name for that controller so it'd be `group_group_membership_path`?